### DISCUSSION TITLE
group 2 discussions

### DISCUSSION THREAD

**Discussion by aviv82** [lab-antwerp-1/group-2#1] [General]
group 2!

we have a lot of work ahead of us, and not a lot of time. we can do a great job as long as we are organized.
i am starting this discussion to share a few resources to help us do it well.

also we can check in here and let each other know what we are working on and how its coming along.

here are a couple of useful links on setting ssh keys git, github and branching:

https://github.com/lab-antwerp-1/home/discussions/6 <-- how to set up ssh key.
https://www.youtube.com/watch?v=RGOj5yH7evk&t=3391s <-- this video is long, but super helpful. especially the part about branching.

a helpful video on how to write markdown:
https://www.youtube.com/watch?v=eJojC3lSkwg&t=456s

we will have a meeting wednesday (6th) morning and hopefully have the main README.md done by then, so we can focus the rest of the time on our individual intro files.

**Comment by HansMbua** [comment]
okay bro I will soo available

**Comment by HansMbua** [comment]
thank you for the YouTube link, check out a free course on git not much but just the essential : https://amigoscode.com/courses/enrolled

**Reply by HansMbua** [reply]
I just saw the read me you created can you create a pull request so  we can pull and review or dose everyone agree ?

**Reply by Mervatabuamro** [reply]
I saw it two but I find it different than the other teams  " the part after who we are" is that okay?

**Reply by HansMbua** [reply]
hello
